src/bin/test_ioctls.rs

```diff
@@ -63,29 +63,31 @@ fn guest_add_mem(pid: Pid) -> Result<()> {
     println!("--");
 
     // add memslot
-    const SLOT_LEN: usize = 0x10;
-    let hv_memslot = tracee.alloc_mem::<[u8; SLOT_LEN]>()?;
+    let slot_len = 4096; // must be page aligned
```

Comment:
Does mmap not always page align? Or does it need to be at least one page big?

Reply:
it needs to be `n*PAGESIZE` big. I improved the comment.